src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -687,119 +703,118 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
         /// </summary>
         private async Task GetAliases()
         {
-            if (!this.areAliasesLoaded)
+            if (_areAliasesLoaded)
             {
-                try
-                {
-                    RunspaceHandle runspaceHandle =
-                        await this.powerShellContext.GetRunspaceHandle(
-                            new CancellationTokenSource(DefaultWaitTimeoutMilliseconds).Token);
+                return;
+            }
 
-                    CommandInvocationIntrinsics invokeCommand = runspaceHandle.Runspace.SessionStateProxy.InvokeCommand;
-                    IEnumerable<CommandInfo> aliases = invokeCommand.GetCommands("*", CommandTypes.Alias, true);
+            try
+            {
+                RunspaceHandle runspaceHandle =
+                    await _powerShellContext.GetRunspaceHandle(
+                        new CancellationTokenSource(DefaultWaitTimeoutMilliseconds).Token);
+
+                CommandInvocationIntrinsics invokeCommand = runspaceHandle.Runspace.SessionStateProxy.InvokeCommand;
+                IEnumerable<CommandInfo> aliases = invokeCommand.GetCommands("*", CommandTypes.Alias, true);
 
-                    runspaceHandle.Dispose();
+                runspaceHandle.Dispose();
 
-                    foreach (AliasInfo aliasInfo in aliases)
+                foreach (AliasInfo aliasInfo in aliases)
+                {
+                    if (!_cmdletToAliasDictionary.ContainsKey(aliasInfo.Definition))
+                    {
+                        _cmdletToAliasDictionary.Add(aliasInfo.Definition, new List<String>() { aliasInfo.Name });
```

Comment:
nit: don't need the `()` :)  - actually if `.Add` can accept an array we can refactor this to:

`new[] { aliasInfo.Name }` I believe

Reply:
Personally I don't like omitting the `()` on the basis that `()` syntactically indicates a "call" and is consistent with other constructor usage, but happy to do whatever there.

Has to be a `List` though I think, since in the `else` branch we are adding to that list. The logic is basically:
```
if dictionary does not have entry:
    add new dictionary entry with singleton list
else:
    add new item to existing list
```